tests/abi_roundtrip.py

```diff
@@ -67,13 +65,6 @@ def bool_comp(x: abi.Bool, *, output: abi.Bool):
     def max_int(cls, bit_size):
         return (1 << bit_size) - 1
 
-    def transaction_comp_factory(self) -> pt.ABIReturnSubroutine:
```

Comment:
Though pyteal was getting generated, it would not align with the original design of the roundtrip test. So am proposing it's removal.